src/client/components/ProductDetail/ProductDetail.component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ProductDetail.styles.css';
+import Button from '../Button/Button.component';
+
+export default function ProductDetail({
+  product,
+  quantityCount,
+  setQuantityCount,
+}) {
+  function decreamentQuantityCount() {
+    setQuantityCount((prevCount) => prevCount - 1);
+  }
+
+  function increamentQuantityCount() {
+    setQuantityCount((prevCount) => prevCount + 1);
+  }
+
+  function handleQuantityCountChange(e) {
+    e.preventDefault();
+    setQuantityCount(e.target.value);
+  }
+
+  const { name, price, picture } = product;
+  return (
+    <div className="productDetail">
+      <article className="productDetail-container">
+        <section className="image-container">
+          <img src={picture} alt={name} />
+        </section>
+        <section className="info-container">
+          <h3>{name}</h3>
+          <div className="info-container-stripes">
+            <div className="info-container-stripe">
+              <p>2lt pot </p>
+              <p>DKK &nbsp;{price}</p>
+            </div>
+            <div className="info-container-stripe">
+              <p>Quantity: &nbsp;</p>
+              <button
+                type="button"
+                className="quantityButton"
+                onClick={decreamentQuantityCount}
+              >
+                -
+              </button>
+              <input
+                type="text"
+                value={quantityCount}
+                onChange={handleQuantityCountChange}
+              />
+              <button
+                type="button"
+                className="quantityButton"
+                onClick={increamentQuantityCount}
+              >
+                +
+              </button>
+            </div>
+          </div>
+          <Button
+            className="productDetail-button"
+            type="button"
+            onClick={' '}
```

Comment:
I'm guessing another PR will add logic for how to add the product to the cart?

Reply:
I guess when we click on that button it should display up a model component.